OJP/OJP_JourneySupport.xsd

```diff
@@ -63,6 +63,11 @@
 		</xs:annotation>
 		<xs:sequence>
 			<xs:element ref="JourneyRef"/>
+			<xs:element name="PublicCode" type="xs:string" minOccurs="0">
```

Comment:
Why is `OperatorName` below added as `type="xs:normalizedString` and `PublicCode` here as `type="xs:string"`? Wouldn't `type="xs:normalizedString` make sense here as well?

https://www.w3schools.com/xml/schema_dtypes_string.asp

Reply:
changed